test/e2e/recurly.test.js

```diff
@@ -174,7 +174,34 @@ describe('Recurly.js', async () => {
       }, sel);
 
       assert.strictEqual(err, null);
-      assertIsAToken(token, expectedType="becs_bank_account");
+    });
+  });
+
+  describe('fraud', async function () {
+    beforeEach(init());
+    it('attaches an input with fraud session ID', async () => {
+      await browser.executeAsync((done) => {
```

Comment:
Can't use `configureRecurly` for this because I need to pass in the `formElement` from the browser context